keras_cv/visualization/draw_bounding_boxes.py

```diff
@@ -117,11 +128,26 @@ def draw_bounding_boxes(
                 cv2.putText(
                     image,
                     label,
-                    (x, y - 10),
+                    (x, y),
                     cv2.FONT_HERSHEY_SIMPLEX,
                     font_scale,
                     color,
                     text_thickness,
                 )
         result.append(image)
     return np.array(result).astype(int)
+
+
+def _find_text_location(x, y, font_scale, line_thickness, outline_factor):
```

Comment:
Just to sanity check -- have you tested this on a few potential edge cases? I can't totally parse the logic, but the demos look good.

Reply:
I have, I tried fiddling with thicknesses and font scales and outline factors quite a bit.  I've failed to find a failure case, but if there is one we can submit new cases to handle it.